stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -996,7 +996,7 @@ public struct Set<Element : Hashable> :
   ///
   /// In the following example, the `bothNeighborsAndEmployees` set is made up
   /// of the elements that are in *both* the `employees` and `neighbors` sets.
-  /// Elements that are only in one or the other are left out of the result of
+  /// Elements that are in only one or the other are left out of the result of
```

Comment:
In a phrase "in only one of them", I would agree, but in this case, where its "only in one or the other", the previous version reads more precise to me... Maybe I'm wrong, just pointing this out in case you misread the sentence.